assets/src/styles/blocks/Covers/TakeActionCovers.scss

```diff
@@ -146,12 +146,17 @@
 
     .btn {
       display: block;
-      background-color: var(--cover-card-btn-background-color, $orange);
-      border-color: var(--cover-card-btn-border-color, $orange);
+
+      --block-covers--card-btn-- {
```

Comment:
I changed the name of this variable (and some others), but only after checking that there is no usage of it among any other repo of the Greenpeace Github organisation. (e.g. https://github.com/search?q=org%3Agreenpeace+var(--cover-card-heading-color&type=code)